### PR TITLE
Test cases file

### PR DESCRIPTION
**Pull request by rafalszmit**
Test cases file to review

### CODE REVIEW

File: metricsTests/testCases.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+3. Should return correct data when 'until' is specified using <timestamp> format
+	1. Open config file
+	2. Fill from and until parameters with values: 
+		from=<timestamp>
+		until=<timestamp>
+	3. Fill all other parameters with default values
+	4. Execute program
+	5. Compare results with expected results
+
+4. Should return correct data when executed for previous month
```
Comment (jacekokrojek) [inline metricsTests/testCases.md]: I wonder why we have this test if we have test 3. which is almost the same

File: metricsTests/testCases.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		until=now
+	3. Fill all other parameters with default values
+	4. Execute program
+	5. Compare results with expected results
+ 
+2. Should return correct data when 'from' is specified using -<day>d format
+	1. Open config file
+	2. Fill from and until parameters with values: 
+		from=-7d
+		until=<timestamp>
```
Comment (jacekokrojek) [inline metricsTests/testCases.md]: Please correct <timestamp> entry - it is wrongly presented by Markdown engine

File: metricsTests/testCases.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+		until=1527804059
+	3. Fill all other parameters with default values
+	4. Execute program
+	5. Compare results with expected results
+
+5. Should return correct data when executed against not default grafana server
+	1. Open config file
+	2. Fill website parameters
+		scheme=http
+		domain=176.119.59.95
```
Comment (jacekokrojek) [inline metricsTests/testCases.md]: I think this is ip address of the default server

File: metricsTests/testCases.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+	4. Execute program
+	5. Compare results with expected results
+
+6. Should return correct data when executed for only 1 metric
+	1. Open config file
+	2. Fill amoutOfTargets parameter with value: 
+		amoutOfTargets=1
+	3. Fill all other parameters with default values
+	4. Execute program
+	5. Compare results with expected results
```
Comment (jacekokrojek) [inline metricsTests/testCases.md]: I think here the expected result should me more precise and says Only one metric is visible in the data file

File: metricsTests/testCases.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+### Valid inputs
+
+1. Should return correct data when executed for last week
+	1. Open config file
+	2. Fill from and until parameters with values: 
+		from=-7d
+		until=now
+	3. Fill all other parameters with default values
+	4. Execute program
+	5. Compare results with expected results
```
Comment (jacekokrojek) [inline metricsTests/testCases.md]: I think that expected result is too general, you should state that for example tester should compare results with values visible on Grafana UI and should calculate stdev.

File: metricsTests/testCases.md (hunk 1 above)
Comment (rafalszmit) [reply metricsTests/testCases.md]: True. One of them should be removed.

**Pull request merged**